release-notes.md

```diff
@@ -4,7 +4,7 @@
 
 **Places Monitor Launch Extension 2.0.0**
 
-* Update Android and iOS installation instruction to Places Monitor 2.0
+* Updated Android and iOS installation instruction to Places Monitor 2.0.
```

Comment:
Sorry, now that I read it, I should've also mentioned the following:

Please reword to read as follows:

"Updated the Android and iOS installation instructions for the Places Monitor 2.0."